twilio-flex-webchat/src/components/VideoButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import * as FlexWebChat from "@twilio/flex-webchat-ui";
+
+const VideoButton = ({ onClick }) => {
+  return (
+    <span className="video-icon-container" onClick={onClick}>
+      <FlexWebChat.Icon icon="Video" />
+      <style jsx="true">{`
```

Comment:
OOC, I see sometimes we do `jsx="true"`, and other times we don't. I'm trying to Google for the answer but I'm having a tough time searching the right terms: why should we use it? Just for my learning, thank you!

Reply:
Ah thanks for noticing. This was a styled jsx issue I meant to fix before and forgot. (It wasn't compiling correctly.) The short answer is you shouldn't ever use it! 😂